service/push-consumer/service.go

```diff
@@ -17,7 +18,10 @@ type Service struct {
 	health   *health.Core
 	config   *Config
 	registry registry.IRegistry
+	le       *leaderelection.LeaderElector
```

Comment:
nit: `leaderelection.LeaderElector` has repetitive words and go discourages this, how about leaderelection.Controller or leaderelection.Commisioner or something else?

Reply:
renaming to Candidate